src/System.Diagnostics.Tools/src/System.Diagnostics.Tools.csproj

```diff
@@ -9,12 +9,17 @@
     <AssemblyName>System.Diagnostics.Tools</AssemblyName>
```

Comment:
FYI: A lot of these project renames will break our internal builds as they are referenced by our wrapper projects. So you will need to handle that when this is merged.